src/renderer/views/Watch/Watch.sass

```diff
@@ -65,6 +65,8 @@
     grid-column: 1
 
   .infoArea
+    position: sticky
```

Comment:
When the window is narrow (e.g. phones and tablets) the `.infoArea` and `.sidebarArea` are vertically stacked instead of being next to each other. This sticky should only be in effect when the `.infoArea` and `.sidebarArea` are next to each other as it causes weird overlay scrolling when those components are vertically stacked.

![weird-scrolling](https://user-images.githubusercontent.com/48293849/171447729-e9ffb3b2-f69d-49fc-9b96-825734449418.jpg)

Reply:
Well spotted, fixed!